client/src/components/common/applicant-info-3.vue

```diff
@@ -7,7 +7,6 @@
           <v-text-field :messages="messages['contact']"
                         :value="applicant.contact"
                         @blur="messages = {}"
-                        @focus="messages['contact'] = 'Contact Name (Optional)'"
```

Comment:
Out of curiousity, what did this do?

Reply:
it show up in the position of validation error message and it is showing as red even after. ux issue raised by tracey
![image](https://user-images.githubusercontent.com/64869009/104964447-4a808a00-5991-11eb-9358-54e4cd28da73.png)

Reply:
Yeah, that was gross. I'm glad it's gone now :)